man/org.freedesktop.systemd1.xml

```diff
@@ -1685,6 +1685,8 @@ node /org/freedesktop/systemd1/unit/avahi_2ddaemon_2eservice {
       readonly b IgnoreOnIsolate = ...;
       @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
       readonly b NeedDaemonReload = ...;
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("false")
```

Comment:
we already have a UnitLoadState and a UnitActiveState and and a UnitFileState, it's quite confusing. But these three are actual states as in state machine. But this new StateFlags field isn't realllllllyyy a state in that sense, It's more a marker a flag, but i wouldn't associate the word "state" with it.

Naming things is always the hardest part...

Maybe: "MarkerFlags", "NeedFlags", "IndicatorFlags" or so?